src/content/reference/gsctl/upgrade-cluster.md

```diff
@@ -28,9 +28,18 @@ To prevent the interactive confirmation, you can use the `--force` flag. This wi
 gsctl upgrade cluster f01r4 --force
 ```
 
+If you want to upgrade to a specific version, you can use the `--release` flag:
+
+```nohighlight
+gsctl upgrade cluster f01r4 --release 13.0.0
+```
```

Comment:
What about adding a notice (`__warning__:`) saying that skipping upgrades can cause unexpected impact?

Reply:
If skipping upgrades is problematic, we should block them using admission controllers.
We do that in azure

Reply:
+1. Our admission controllers should prevent skipping majors. No need to raise discomfort here.